netdeploy/remote/deployedNetwork.go

```diff
@@ -283,9 +338,424 @@ func (cfg DeployedNetwork) BuildNetworkFromTemplate(buildCfg BuildConfig, rootDi
+	bootstrappedNet.nApplications = fileCfgs.GeneratedApplicationCount
+	//bootstrappedNet.nApplications = 0
+	bootstrappedNet.txnState = protocol.PaymentTx
+	bootstrappedNet.roundTxnCnt = fileCfgs.RoundTransactionsCount
+	bootstrappedNet.round = basics.Round(0)
+
+	params := config.Consensus[protocol.ConsensusCurrentVersion]
+	minAccounts := accountsNeeded(fileCfgs.GeneratedApplicationCount, fileCfgs.GeneratedAssetsCount, params)
+	nAccounts := fileCfgs.GeneratedAccountsCount
+	if minAccounts > nAccounts {
+		bootstrappedNet.nAccounts = minAccounts
+	} else {
+		bootstrappedNet.nAccounts = nAccounts
+	}
+
+	accounts[poolAddr] = basics.MakeAccountData(basics.NotParticipating, rewardsPool.State.MicroAlgos)
+	accounts[sinkAddr] = basics.MakeAccountData(basics.NotParticipating, feeSink.State.MicroAlgos)
+	//fund src account with enough funding
+	bootstrappedNet.fundPerAccount = basics.MicroAlgos{Raw: uint64(bootstrappedNet.nAssets) * params.MinBalance * 2}
+	totalFunds := srcWallet.State.MicroAlgos.Raw + bootstrappedNet.fundPerAccount.Raw*bootstrappedNet.nAccounts + bootstrappedNet.roundTxnCnt*fileCfgs.NumRounds
+	accounts[src] = basics.MakeAccountData(basics.Online, basics.MicroAlgos{Raw: totalFunds})
+
+	bootstrappedNet.poolAddr = poolAddr
+	bootstrappedNet.sinkAddr = sinkAddr
+
+	//gensis block
+	initState, err := generateInitState(accounts, &bootstrappedNet)
+	if err != nil {
+		return err
+	}
+	localCfg := config.GetDefaultLocal()
+	localCfg.Archival = true
+	localCfg.CatchpointTracking = -1
+	localCfg.LedgerSynchronousMode = 0
+	log := logging.NewLogger()
+	l, err := ledger.OpenLedger(log, genesisFolder+"/bootstrapped", false, initState, localCfg)
+	if err != nil {
+		return err
+	}
+	l.WaitForCommit(0)
+
+	//create accounts, apps and assets
+	prev, _ := l.Block(l.Latest())
+	err = generateAccounts(src, fileCfgs.RoundTransactionsCount, prev, l, &bootstrappedNet, params)
+	if err != nil {
+		return err
+	}
+
+	//create more transactions
+	prev, _ = l.Block(l.Latest())
+	for i := uint64(bootstrappedNet.round); i < fileCfgs.NumRounds; i++ {
+		bootstrappedNet.round++
+		blk, _ := createBlock(src, prev, fileCfgs.RoundTransactionsCount, &bootstrappedNet, params)
+		blk.BlockHeader.TimeStamp += int64(crypto.RandUint64() % 100 * 1000)
+		err = l.AddBlock(blk, agreement.Certificate{Round: bootstrappedNet.round})
+		if err != nil {
+			fmt.Printf("Error %s\n", err)
+			return err
+		}
+		prev, _ = l.Block(l.Latest())
+	}
+
+	l.WaitForCommit(bootstrappedNet.round)
+	l.Close()
+
+	localCfg.CatchpointTracking = 0
+	l, err = ledger.OpenLedger(log, genesisFolder+"/bootstrapped", false, initState, localCfg)
+	if err != nil {
+		return err
+	}
+	l.Close()
+
+	fmt.Printf("ends %v\n", time.Now())
+	return nil
+}
+
+func getGenesisAlloc(name string, allocation []bookkeeping.GenesisAllocation) bookkeeping.GenesisAllocation {
+	for _, alloc := range allocation {
+		if strings.ToLower(alloc.Comment) == strings.ToLower(name) {
```

Comment:
move the `strings.ToLower(name)` outside of the loop, so that :
```
name = strings.ToLower(name)
...
     if strings.ToLower(alloc.Comment) == name {
```